requirements.txt

```diff
@@ -1 +1,2 @@
-ladybug-geometry==1.23.19
\ No newline at end of file
+lbt-ladybug==0.25.96
```

Comment:
Shouldn't you only need the geometry library? Why do we need `lbt-ladybug` here?

Reply:
I see that for now, you are using it for tests. In that case, you probably should move this dependency to dev-requirements which get installed for running tests.

Reply:
I was wrong. We actually need the core library to be able to extend the classes.